source/agora/consensus/data/genesis/Test.d

```diff
@@ -128,6 +128,14 @@ public immutable Block GenesisBlock = {
     ],
 };
 
+version (unittest)
+{
+    import std.algorithm;
+
+    static immutable testGenesisBlockFreezeTransaction = GenesisBlock.txs.filter!(tx => tx.type == TxType.Freeze).front;
+    static immutable testGenesisBlockPaymentTransaction = GenesisBlock.txs.filter!(tx => tx.type == TxType.Payment).front;
+}
+
```

Comment:
Let's not repeat the module name in the symbol. GenesisFreezeTransaction (Tx?) should do, shouldn't it ?

Reply:
Moved to block and renamed.